src/Task/Git/CommitMessage.php

```diff
@@ -96,7 +96,18 @@ public function run(ContextInterface $context)
         }
 
         if (count($exceptions)) {
-            return TaskResult::createFailed($this, $context, implode(PHP_EOL, $exceptions));
+            return TaskResult::createFailed(
+                $this,
+                $context,
+                implode(PHP_EOL, $exceptions) . <<<KAWAii
```

Comment:
Other tasks have a `\GrumPHP\Formatter\ProcessFormatterInterface` as dependency in their constructor. Since this task does not run as a proccess, we might need to add a new interface or just a formatter without an interface that can be added to the constructor of the task. This formatter can then be used to style the exceptions. It is interchangeable in the configuration so that you can add your own formatter if you like. Feel free to ask any additiontal questions if the information above is not sufficient.